test/jdk/sun/security/util/Resources/Usages.java

```diff
@@ -75,7 +75,7 @@
             "(?m)rb[ \\n]*\\.getString[ \\n]*\\([ \\n]*\"(.*?)\"\\)");
 
     static Pattern EVENT_OCSP_CRL = Pattern.compile(
-            "Event\\.report\\(\"(.*?)\",");
+            "Event\\.report\\(.*,\"(.*?)\",");
```

Comment:
If you agree to add a whitespace in the calls in OSCP.java and DistributionFetcher.java, you might want to add a whtespace (or `\s*`) here as well.